inspirehep/modules/search/walkers/elasticsearch.py

```diff
@@ -118,11 +118,16 @@ def query(keyword):
 
             fields = self.get_fields_for_keyword(keyword, mode='a')
             if fields == ['authors.full_name', 'authors.alternative_name']:
```

Comment:
This is part of a series of refactoring and design decisions that we need to sit down and make straight. Hopefully part of @jacquerie's future journey in the mapping world.

Meanwhile, :+1: to @jmartinm comment.